crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1159,9 +1181,50 @@ impl fmt::Debug for World {
     }
 }
 
-unsafe impl Send for World {}
 unsafe impl Sync for World {}
```

Comment:
Just so I understand: why do we still want the `Sync` impl for `World`?

Reply:
The entire idea of our world abstraction is that it is accessed in a shared way across multiple threads.